share/sound/MS Basic_License.md

```diff
@@ -1,21 +1,21 @@
-# MuseScore_General.sf2
+ # MS Basic.sf2
 ---
 
 Current version: 0.2  13th May 2020
 
-This is a scaled-down version of **MuseScore_General-HQ.sf2** that replaces some of the larger instruments to save memory and CPU on older PCs. This SoundFont is currently a work-in-progress. Detailed information on presets and sample sources used can be found in "MuseScore_General_Sample_Sources.csv". All instruments without attribution are still using samples from FluidR3Mono.
+This is a scaled-down version of **MS Basic-HQ.sf2** that replaces some of the larger instruments to save memory and CPU on older PCs. This SoundFont is currently a work-in-progress. Detailed information on presets and sample sources used can be found in "MS Basic_Sample_Sources.csv". All instruments without attribution are still using samples from FluidR3Mono.

```

Comment:
In fact this is not actual. MS Basic is a renamed and slightly-modified copy of `MuseScore_General-HQ.sf2`, which was indeed the high-quality soundfont from MS3. @shoogle May I ask you to handle this, and suggest updates for this text?